pkg/util/common.go

```diff
@@ -170,13 +171,13 @@ func CreateSecretVolume(volumeName string, secretName string, defaultMode int) (
 }
 
 // CreatePod will create the pod
-func CreatePod(name string, serviceAccount string, volumes []*components.Volume, containers []*Container, initContainers []*Container, affinityConfigs []horizonapi.AffinityConfig, labels map[string]string) *components.Pod {
+func CreatePod(name string, serviceAccount string, volumes []*components.Volume, containers []*Container, initContainers []*Container, affinityConfigs []horizonapi.PodAffinityConfig, labels map[string]string) (*components.Pod, error) {
```

Comment:
Do we want to support PodAntiAffinity and NodeAffinity as well as hard and soft for pod creation?  Seems like we should.

Reply:
done